webui/react/src/omnibar/Component.module.scss

```diff
@@ -0,0 +1,21 @@
+.base {
+  background-color: rgba(0, 0, 0, 0.7);
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+
+  .bar {
+    left: 50%;
+    min-width: 50rem;
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    width: calc(max(40vw, 300px));
```

Comment:
**style suggestion:** adding a mobile break point so it looks proper on a mobile device:
```
@media only screen and (max-width: $breakpoint-mobile) {
  width: ...
  top: ...
}
```
Haven't played around too much but for mobile, it might be better to have the bar right at the top and have a padding all around both the omnibar and the options that show. The options would be nice if it has a max height of 100vh minus the omnibar height and also scroll if it spills past the screen height.

Reply:
hmm I wouldn't expect mobile users to be using this 🤔  how would they even trigger this? unless we add a special case for them

Reply:
lol that's a great point. So used to supporting all breakpoints, without consideration of how they can actually even trigger it.